src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -133,6 +146,166 @@ function showGiftCardWarningMessage() {
   orderTotalSummaryEl.appendChild(alertContainer);
 }
 
+function attachGiftCardFormListeners() {
+  const giftCardSelect = document.querySelector('#giftCardSelect');
```

Comment:
Can we move these selectors to their own method? We repeat the code to retrieve them (including string ID as argument) throughout this class. Might be more scalable if we have that in a single function that returns the selected UI elements as an object.

Reply:
Great tip! I will do that